src/components/standalone/firewall/nat/CreateOrEditNatRuleDrawer.vue

```diff
@@ -130,22 +148,26 @@ watch(
       })
       firewallConfig.fetch()
       listHostSuggestions()
+      listAvailableDevices()
       sourceAddress.value = ''
       destinationAddress.value = ''
+      advancedSettingsExpanded.value = false
```

Comment:
We usually expand advanced settings if the the user made a change to its default settings. We can do this at line 171 instead, according to `props.currentRule.device`